posthog/settings/temporal.py

```diff
@@ -10,6 +10,9 @@
 TEMPORAL_CLIENT_CERT: str | None = os.getenv("TEMPORAL_CLIENT_CERT", None)
 TEMPORAL_CLIENT_KEY: str | None = os.getenv("TEMPORAL_CLIENT_KEY", None)
 TEMPORAL_WORKFLOW_MAX_ATTEMPTS: str = os.getenv("TEMPORAL_WORKFLOW_MAX_ATTEMPTS", "3")
+MAX_CONCURRENT_WORKFLOW_TASKS: int | None = get_from_env(
+    "MAX_CONCURRENT_WORKFLOW_TASKS", None, optional=True, type_cast=int
+)
```

Comment:
I'm not sure what's the Temporal default (the Python code is `Optional[int]` type, so should be `None`, but obviously that doesn't mean 0 workflow tasks available).

I'm leaving this as `None` to avoid breaking other deployments in case we mess up the default.